playbooks/ansible-cloud/vcenter-appliance/files/bootstrap.yaml

```diff
@@ -33,10 +33,10 @@
         name: zuul
         password: "{{ __password | password_hash('sha512') }}"
 
-- hosts: vmware-vcsa-6.7.0
+- hosts: vcenter
```

Comment:
We likely want this to be the group name, so appliance, right? As we could have more then 1 vcenter node.

Reply:
I think it's fine since we will have both esxi and vcenter in appliance group. Here we only target the vcenter. And so far we don't have any test with two vcenter.